apps/condo/domains/common/components/tasks/TaskProgress.tsx

```diff
@@ -201,17 +203,28 @@ export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task
 
 const VisibilityControlButton = styled.div`
   position: absolute;
-  right: 30px;
-  top: 20px;
+  right: 20px;
+  top: 19px;
   z-index: 1;
   cursor: pointer;
 `
 
+const ChevronCollapseIconStyle = css`
+  transform-origin: center;
+  transform: rotate(180deg);
+`
+
 const infoIconStyles = css`
   color: ${colors.infoIconColor};
   font-size: 20px;
 `
 
+const TasksProgressTitleStyle = css`
+  font-size: 16px;
```

Comment:
please use font sizes from constants

Reply:
It is a custom component, It does not uses something global.
No need to extract it into constant, because we have only one usage place of that.